eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -135,6 +137,15 @@ function Get-javascript-UpdatedDocsMsToc($toc) {
         )
       }
     }
+
+    if ($services[$i].name -eq 'Cognitive Services') {
```

Comment:
is the idea to do this in all repos?

Reply:
It is not. This function `Get-javascript-UpdateDocsMsToc` is specific to just the JS repo.

Reply:
More context: This is just happening in JS because the JS package for OpenAI is not documentable with learn.microsoft.com's tools at the moment. We won't propagate this change to other repos because documentation for those packages does not have the same issues.